internal/common/metrics/constants.go

```diff
@@ -43,11 +43,16 @@ const (
 	WorkflowTaskNoCompletionCounter     = TemporalMetricsPrefix + "workflow_task_no_completion"
 
 	ActivityPollNoTaskCounter             = TemporalMetricsPrefix + "activity_poll_no_task"
+	NexusPollNoTaskCounter                = TemporalMetricsPrefix + "nexus_poll_no_task"
```

Comment:
I think putting all `Nexus` constants together is a bit easier to read than interleaved with their activity equivalents

Reply:
Should I group the activity and workflow ones too?

Reply:
Nah, I'd just add your stuff separate